CONTRIBUTING.md

```diff
@@ -34,20 +34,20 @@ process for contributions:
 1. Open an issue on the [project
    repository](https://github.com/markdownlint/markdownlint/issues), if
    appropriate
-1. If you're adding or making changes to rules, read the [Development
+2. If you're adding or making changes to rules, read the [Development
```

Comment:
please revert this. Markdown best practices is to use all `1.` so that when you add/reorder, it all just renders properly.

further, this isn't in the description of your PR

Reply:
Addressed by commit [536c772](https://github.com/markdownlint/markdownlint/pull/445/commits/536c7721a80ccc0219ab2725b306539cf020cf37)